x/farming/types/genesis.go

```diff
@@ -37,7 +38,9 @@ func DefaultGenesisState() *GenesisState {
 		[]CurrentEpochRecord{},
 		sdk.Coins{},
 		sdk.Coins{},
-		time.Time{})
+		time.Time{},
+		1,
```

Comment:
We can use a named `const` instead of hard-coded `1`, just like `DefaultNextEpochDays`.